modules/nft/src/lib.rs

```diff
@@ -0,0 +1,229 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::{Decode, Encode};
+use frame_support::{
+	decl_error, decl_event, decl_module, ensure,
+	traits::{Get, IsType},
+};
+use frame_system::ensure_signed;
+use orml_nft::CID;
+use orml_traits::{BasicCurrency, BasicReservableCurrency};
+use orml_utilities::with_transaction_result;
+use primitives::Balance;
+use sp_runtime::{
+	traits::{AccountIdConversion, Zero},
+	ModuleId, RuntimeDebug,
+};
+use sp_std::vec::Vec;
+
+mod mock;
+mod tests;
+
+#[derive(Encode, Decode, Clone, RuntimeDebug, PartialEq, Eq)]
+pub enum ClassProperty {
```

Comment:
use `use enumflags2::BitFlags` to make this a bit flags

example:
https://github.com/paritytech/substrate/blob/596f377d49fede088e0c32791fb72f1343003b0d/frame/identity/src/lib.rs#L281-L311